pom.xml

```diff
@@ -193,6 +193,8 @@
 	    <sonar.coverage.exclusions>**/generated/**</sonar.coverage.exclusions>
 	    <sonar.cpd.exclusions>**/generated/**</sonar.cpd.exclusions>
 
+        <hibernate-validator.version>6.0.10.Final</hibernate-validator.version>
```

Comment:
hibernate-validator needs to be added within `<dependencyManagement>` , using this property as the version.

Consider carefully what version - 6.0.18 seems current. If there's a particular reason to go older, fine -- but generally I'd err towards current levels

It's worth checking dependencies too as at https://mvnrepository.com/artifact/org.hibernate/hibernate-validator - though nothing interesting there.

Reply:
see https://github.com/odpi/egeria/blob/master/developer-resources/Dependency-Management.md